### PR TITLE
update for 0.3.0

### PR DESCRIPTION
**Pull request by elijah-rou**

### CODE REVIEW

File: changelog.mdx [hunk 1]
```diff
@@ -4,6 +4,11 @@ description:
   "We release features and fixes weekly! See some of the changes we have made."
 ---
 
+## 0.3.0
+
+#### Date: 25 November 2022
+- We've switched out infrastructure provider for something more reliable and faster. This means faster deployment, less downtime and roughly half the inference time!
```
Comment (milo157) [inline changelog.mdx:10]: I wouldn't say we switched out our infra provider but rather "We made large changes to our infrastructure which means faster deployment etc"

**Review by milo157 — APPROVED** [review]
Just 1 change on changelog

File: changelog.mdx (hunk 1 above)
Comment (elijah-rou) [reply changelog.mdx:10]: ++

File: changelog.mdx [hunk 2]
```diff
@@ -15,7 +20,7 @@ description:
 ## 0.2.3
 
 #### Date: 11 November 2022
--  Added support for post-processing functions in pipelines!
+-  Added support for post-processing functions in s!
```
Comment (jonoirwinrsa) [inline changelog.mdx:23]: ssss

**Review by jonoirwinrsa — APPROVED** [review]

**Pull request merged**